back/app/controllers/omniauth_callback_controller.rb

```diff
@@ -94,26 +89,23 @@ def auth_callback(verify:, authver_method:)
           @user.save!
           @invite.save!
           SideFxInviteService.new.after_accept @invite
-          signup_success_redirect
+          verify_and_sign_in(auth, @user, verify, sign_up: true)
```

Comment:
I guess the main change to fix the "user created even though verification failed"-issue is here, right?

But it seems that the user is still created above, and that the only thing is that we don't set the token. Am I understanding that correctly? If so, should we even create this user if the verification fails?

Reply:
Deleting the user later if it's been created, but in this case we need to rollback the invite state. Should probably to the refactor I deliberately haven't done to verify first before getting the user